src/runtime/storage/pouchdb/pouch-db-variable.ts

```diff
@@ -141,19 +141,22 @@ export class PouchDbVariable extends PouchDbStorageProvider {
    * @return a promise containing the variable value or null if it does not exist.
    */
   async get(): Promise<ValueStorage> {
-    const value = await this.getStored();
-
-    if (this.referenceMode && value) {
-      try {
-        await this.ensureBackingStore();
-        return await this.backingStore.get(value.id);
-      } catch (err) {
-        console.warn('PouchDbVariable.get err=', err);
-        throw err;
+    try {
+      const value = await this.getStored();
+      if (this.referenceMode && value) {
+        try {
+          await this.ensureBackingStore();
+          return await this.backingStore.get(value.id);
+        } catch (err) {
+          //console.warn('PouchDbVariable.get err=', err);
+          throw err;
```

Comment:
do we need both try-catch-es?

Reply:
I don't know exactly, I tried to minimize changes to the original semantics. The inner try was already there, the outer try protects against exceptions from `getStored`.

Reply:
l should have added a comment about the disabled warning, will do that.